test/BridgeExecutorBase.spec.ts

```diff
@@ -399,6 +400,46 @@ describe('BridgeExecutorBase', async function () {
       expect((await bridgeExecutor.getActionsSetById(0)).executed).to.be.equal(true);
     });
 
+    it('Queue and fail to execute an actions set with insufficient value', async () => {
+      const greeter = await new Greeter__factory(user).deploy();
+      const NEW_MESSAGE = 'hello';
+
+      const { data } = encodeSimpleActionsSet(greeter.address, 'setMessage(string)', [NEW_MESSAGE]);
+      const value = (await ethers.provider.getBalance(bridgeExecutor.address)).add(1);
+      await expect(bridgeExecutor.queue(
+        data[0] as string[],
+        [value],
+        data[2] as string[],
+        data[3] as string[],
+        data[4] as boolean[]
+      )).to.not.be.reverted;
```

Comment:
Here is the only one use of `to.not.be.reverted`.
The others tests just do `expect(await tx)` , and additionally check for event emissions

Reply:
Agree